studio/schemas/documents/siteSettings.ts

```diff
@@ -1,10 +1,13 @@
 import { defineType, defineField } from "sanity";
```

Comment:
Studio will crash when moving between schema, becuase documents can still be present with the old document type. Deprecating the document types and fields (like Analytics) avoids this.

Reply:
for cases such as these, don't add deprecated but make a separate PR to merge once studio has been updated.

Reply:
This is just extra documentation before we remove it completely, but sure I can remove the deprecation notice.

Reply:
The reason why I want to avoid adding deprecated is because it's so easy to overlook as a task to remove. That and adding deprecated signals to other devs that yes it's currently deprecated, but still not something we want to remove completely. It also doesn't tell me as a reviewer what your intent here is, especially when I don't see a mention of a related task or PR